src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -221,11 +221,16 @@ private static String urlEncodePair(String k, String v) throws UnsupportedEncodi
 		return conn;
 	}
 
-	private static javax.net.ssl.HttpsURLConnection createPostConnection(String url, String query, String apiKey) throws IOException {
+	private static javax.net.ssl.HttpsURLConnection createPostConnection(String url, String query, String apiKey, Boolean isRating) throws IOException {
 		javax.net.ssl.HttpsURLConnection conn = createEasyPostConnection(url, apiKey);
 		conn.setDoOutput(true);
 		conn.setRequestMethod("POST");
-		conn.setRequestProperty("Content-Type", String.format("application/x-www-form-urlencoded;charset=%s", CHARSET));
+		if (!isRating) {
+			conn.setRequestProperty("Content-Type", String.format("application/x-www-form-urlencoded;charset=%s", CHARSET));
+		}
+		else{
+			conn.setRequestProperty("Content-Type", String.format("application/json"));
```

Comment:
Yea, its because the request has to be made in json for Rating calls. If you have any thoughts on how to make this better definitely let me know.